packages/components/src/MultiSelect/_docs/MultiSelect.mdx

```diff
@@ -12,7 +12,11 @@ import * as MultiSelectStories from "./MultiSelect.stories"
   className="!mb-8"
 />
 
-<KaioNotification />
+<StoryInlineNotification
```

Comment:
Maybe instead of being generic, we should the content part of the component? `<SbNotificationUpcomingRelease>` or something?

Reply:
We could, but we may other use cases outside upcoming release warnings / statements, maybe just `SbNotification`?

Reply:
In that situation, we can make another component. This particular notification has a very reusable nature to it for any upcoming components. If the message was very specific to the component, it would be a little more justified, but it may not even be the same problem.

Reply:
I suppose my concern is more around recreating similar components that are, in all but name, an Inline Notification for storybook. That said if there is a high likelihood that we will need more of these in the future and they these would only be used with upcoming releases I don't have an opposition scoping the name

Reply:
Also lets move any more comments over to this over to the [PR](https://github.com/cultureamp/kaizen-design-system/pull/4394#discussion_r1414843236), since this is unrelated to removal work done in this PR :)